src/Loop.php

```diff
@@ -241,7 +241,8 @@ public static function disable($watcherId)
     }
 
     /**
-     * Cancel a watcher. This marks the watcher identifier as invalid.
+     * Cancel a watcher. This will detatch the event loop from all resources that are associated to the watcher. After this
+     * operation the watcher is permanently invalid unless reissued for a different event.
```

Comment:
The driver docs state the identifier is unique, thus it will not be reissued. So, just drop that part " unless reissued for a different event".